src/app/config/config.service.ts

```diff
@@ -47,14 +47,24 @@ export class ConfigService {
     this.initialized = true;
   }
 
+  public generateHttpHeadersForConfig(octoprint: Octoprint): HttpHeaders {
```

Comment:
```suggestion
  public generateHttpHeadersForGivenInstance(octoprint: Octoprint): HttpHeaders {
```

Might make this a little clearer :)